src/components/PostDetail.server.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { marked } from "marked";
+import getPosts from "../server/posts";
+
+export default async function PostDetail({ permalink }) {
+  const list = await getPosts();
+  const post = list.filter((post) => post.permalink === permalink)[0];
+  const html = marked(post.content);
+
+  return <p dangerouslySetInnerHTML={{ __html: html }}></p>;
```

Comment:
this component simply renders intrinsic element of `<p/>`, 
it is more complex when there are nested function components(both server and client). we'll support it in following episodes